uitools/import/Esri/ArcGISRuntime/Toolkit/AuthenticationView.qml

```diff
@@ -47,6 +47,10 @@ import QtQuick.Controls 2.12
         \c{QtWebView::initialize()} immediately after the \c{QGuiApplication} 
         instance is created. See \l{https://doc.qt.io/qt-5/qtwebview-index.html}
         {Qt WebView}.
+
+  \image docs/authenticationview_qml.png authentication_view
+  Example code:
+  \snippet qml_quick/src/demos/AuthenticationViewDemoForm.qml Set up Authentication View QML
```

Comment:
One thing @ldanzinger  asked for was some more explicit text to draw attention to the fact that this example code is QML only. 

E.g.

```
Example of usage in th QML API (C++ API may differ):
```